astrokat/observatory.py

```diff
@@ -6,9 +8,30 @@
 from simulate import user_logger, setobserver
 from utility import katpoint_target, lst2utc
 
+
 try:
     import katconf
-except ImportError:
+    # Set up configuration sourcei
+    config_path = '/var/kat/config'
+    node_file = '/var/kat/node.conf'
+    settings = {}
```

Comment:
All the variables used in this code look like temporary ones, but they will appear in the `observatory` namespace, which could be confusing.  Could prefix with `_`, but probably better to put all this in a private function.